globalstate.py

```diff
@@ -0,0 +1,182 @@
+"""
+Entrypoint for cooperative packages sharing a common global loop.
+
+A minimal plugin looks like this:
+
+```python
+from typing import Optional
+import sublime_asyncio
+import sublime_plugin
+import asyncio
+
+
+async def exit_handler() -> None:
+    print("FooAsync: shutting down lots of asynchronous processes...")
+    await asyncio.sleep(1.0)
+    if _task:
+        _task.cancel()
+    print("Done shutting down")
+
+
+async def run() -> None:
+    try:
+        while True:
+            print("Hello from FooAsync coroutine")
+            await asyncio.sleep(1.0)
+    except asyncio.CancelledError:
+        print("FooAsync got cancelled")
+
+
+_exit_handler_id = 0
+_task: Optional[asyncio.Task] = None
+
+
+def plugin_loaded() -> None:
+    global _exit_handler_id
+    _exit_handler_id = sublime_asyncio.acquire(exit_handler)
+
+    def store_task(task: asyncio.Task) -> None:
+        global _task
+        _task = task
+
+    sublime_asyncio.run_coroutine(run(), store_task)
```

Comment:
`run_coroutine` seems to take one argument.

Reply:
nicely spotted :)